jax/experimental/jax2tf/jax_export.py

```diff
@@ -211,10 +177,11 @@ def serialize_native(fun_jax: Callable,
       in_shardings=lowered.compile_args["in_shardings"],
       out_shardings=lowered.compile_args["out_shardings"],
       lowering_platform=lowering_platform or default_jax_backend(),
-      mlir_module=mlir_module,
+      strict_checks=strict_checks,
       mlir_module_serialized=mlir_module_serialized,
       module_kept_var_idx=module_kept_var_idx,
-      xla_call_module_version=xla_call_module_version)
+      xla_call_module_version=xla_call_module_version,
+      _get_vjp=lambda exported: _export_native_vjp(wrapped_fun_jax, exported))
 
 
 def add_dim_arg_computation(module: mlir.ir.Module,
```

Comment:
My understanding is that this is internal API within jax_export API. can we rename "_add_dim_arg_computation"?
Same question for other functions name.

Reply:
Done. 

This is not yet as important because jax_export is private to jax2tf. But is still a good idea.